docs/references/storage/create-file.md

```diff
@@ -1 +1,7 @@
-Create a new file. The user who creates the file will automatically be assigned to read and write access unless he has passed custom values for read and write arguments.
\ No newline at end of file
+Create a new file. The user who creates the file will automatically be assigned to read and write access unless they have passed custom values for read and write arguments.
+
+Larger files should be uploaded using multiple requests with the [content-range](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Range) header to send a partial request with a maximum supported chunk of `2MB`. The `content-range` header values should always be in bytes.
```

Comment:
Was it 5 or 2 MB?

Reply:
Ongoing, as S3 only supports at least 5 MB chunks, we might rethink this. Right now the payload size is 5MB, but chunk has to be smaller to accomodate other data in the request